docs/ensembl-help/using-ensembl/getting-ensembl-data/tools-and-apis/graphql/getting-started.md

```diff
@@ -0,0 +1,272 @@
+---
+slug: accessing-graphql
+title: Accessing Ensembl GraphQL services
+description: How to access the GraphQL services provided by Ensembl
+related_articles:
+  - href: ./using-graphql.md
+---
+# Accessing Ensembl GraphQL services
+
+The Ensembl GraphQL services can be accessed [here](/data/graphql).
+
+## Schemas and documentation
+If you wish to interrogate the service and explore the documentation, [a GraphQL playground](/data/graphql) can be accessed via your browser.
+
+## Genomes
+
+The data within a GraphQL service is represented as a graph. The way in which the Ensembl data are logically grouped together is by genome.  Genomes are defined to be a collection of datasets and related annotation for a given assembly.  Some assemblies will have a new genome generated for each release whilst other assemblies will have genomes which span multiple releases.
+
+In many of queries, there is a parameter for `genome_id`.  This identifier (a [UUID]("https://www.itu.int/ITU-T/recommendations/rec.aspx?rec=11746&lang=en")) is used to scope queries to the data belonging to a given genome.
+
+To get a `genome_id` you will need to provide the query below a lookup `keyword`, and optionally a `release_number` (if `release_number` is left blank, it will default to the latest genomes).  
+
+The `keyword` can be one of the following:
+- [ToLID](https://id.tol.sanger.ac.uk/) (e.g. aRanTem1)
+- Assembly accession id (e.g. GCA_000001405.28)
+- Taxon ID (e.g. 6239)
+- Scientific name (e.g. Triticum aestivum)
+- Parlance name (e.g. human)
+
+
+Currently partial matches are *not* supported.
+
+For example:
+
+#### Query
+```
+query {
+  genomes(
+    by_keyword: {
+      keyword:"Triticum aestivum",
+      release_version: 108  # Optional
+    }
+  ) {
+    genome_id
+    assembly_accession
+    scientific_name
+    release_number
+  }
+}
+```
+
+#### Response
+```json
+{
+  "data": {
+    "genomes": [
+      {
+        "genome_id": "a73357ab-93e7-11ec-a39d-005056b38ce3",
+        "assembly_accession": "GCA_900519105.1",
+        "scientific_name": "Triticum aestivum",
+        "release_number": 108
+      }
+    ]
+  },
+  "extensions": {
+    "execution_time_in_seconds": 0.11
+  }
+}
+```
+
+The `genome_id` can then be used in subsequent queries.  
+
+## Examples 
+
+## Find a Gene by a symbol
+
+Purpose: Find the Ensembl identifier for a Gene with a specified symbol within a genome.
+
+First we need to find the `genome_id`:
+
+It can be done either by searching using `by_keyword` (as explained above) or `by_assembly_accession_id`.
+
+#### Query
+```
+query {
+  genomes(
+    by_assembly_accession_id: {
+      assembly_accession_id:"GCA_000001405.28"
+    }
+  ) {
+    genome_id
+  }
+}
+```
+
+#### Response
+```json
+{
+  "data": {
+    "genomes": [
+      {
+        "genome_id": "<genome_id>"
+      }
+    ]
+  },
+  "extensions": {
+    "execution_time_in_seconds": 0.1
+  }
+}
+```
```

Comment:
The `extensions` is debug information that is superfluous and is not for public consumption. I suggest removing it from the description of the response payload.